docs: #1308 update for alter partitions with kafka api (#1309)
redpanda-data/documentation: modules/manage/pages/cluster-maintenance/manage-throughput.adoc

= Manage Throughput
:description: Manage the throughput of Kafka traffic with configurable properties.
:page-categories: Management, Networking
// tag::single-source[]

> Redpanda supports applying throughput throttling on both ingress and egress independently, and allows configuration at the broker and client levels. The purpose of this is to prevent unbounded network and disk usage of the broker by clients.

Reviewer: ```suggestion
Redpanda supports throughput throttling on both ingress and egress independently, and allows configuration at the broker and client levels. This helps prevent clients from causing unbounded network and disk usage on brokers.
```

. Quota configured through the Kafka API for an exact match on `client_id`
. Quota configured through the Kafka API for a prefix match on `client_id`

> . Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`-deprecated in v24.2) for a prefix match on `client_id`

Reviewer 2: @micheleRP this should not be in cloud. Anything related to cluster configs just skip.  

This includes both this stuff and  the whole 'migration' section below. There's nothing to migrate from in cloud since they never had access to the old cluster config based quotas

Author: fixed [here](https://deploy-preview-1309--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/cluster-maintenance/manage-throughput/#client-throughput-limits)
